pna_control/pna_control.py

```diff
@@ -80,8 +94,8 @@ def pna_setup(pna,
         # print(f'cal_cmd: {cal_cmd}')
         pna.write(cal_cmd)
 
-        gpoints = pna.query(f'SENSe1:SWEep:POINts?')
-        assert int(gpoints) == points, f'VNA points ({gpoints}) != {points}.'
+        # gpoints = pna.query(f'SENSe1:SWEep:POINts?')
```

Comment:
Should these be deleted?

Reply:
Yes, I put them there when I was debugging the calibration option, but I do not think they are necessary.